### ISSUE TITLE
Make a copy of source data instead of bagging-in-place

### ISSUE THREAD

**Issue by nkrabben** [UAlbanyArchives/mailbagit#227]
**Is your feature request related to a problem? Please describe.**
I prefer to make copies in my workflow, e.g. step 1. transferred package, step 2. make copy, step 3. transform copy

**Describe the solution you'd like**
I would like the source file to be left in place
```
mailbagit source_data -keep ...
ls source_data
> source_data
```

**Describe alternatives you've considered**
We could make the copy outside of the tool
`cp source_data backup/ | mailbagit source_data ...`

**Comment by gwiedeman** [comment]
Draft implementation in [moving_options-226-227](https://github.com/UAlbanyArchives/mailbagit/tree/moving_options-226-227). Just needs fixing tests, more testing with companion files, and updated docs.

**Comment by gwiedeman** [comment]
Implemented by #229
